platforms/blabsy/src/components/view/view-tweet-stats.tsx

```
@@ -10,106 +10,111 @@ import { UserCards } from '@components/user/user-cards';
-          )}
-        </div>
-      )}
-    </>
-  );
+                                        index === 0 && 'cursor-not-allowed'
+                                    )}
+                                    key={title}
+                                    onClick={
+                                        type ? handleOpen(type) : undefined
+                                    }
+                                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)